tests/test_scorers.py

```diff
@@ -5,29 +5,27 @@
 
 
 def load_points_csv(filepath):
-    with open(Path(__file__).parent / filepath, 'r') as f:
+    with open(Path(__file__).parent / filepath, "r") as f:
```

Comment:
Just for the future - make style changes in separate PRs

Reply:
ok